tox.ini

```diff
@@ -449,6 +451,25 @@ commands =
     ; pytest --nbmake tutorials/pandas-cookbook -p no:randomly -vvvv
 
 
+[testenv:single_container.launch]
+description = Launch a single backend container using the dockerfile
+changedir = {toxinidir}/packages
+allowlist_externals =
+    bash
+commands =
+    bash -c 'tox -e single_container.destroy'
+    bash -c 'docker build -f grid/backend/backend.dockerfile . -t openmined/grid-backend:local-dev'
+    bash -c 'docker run -d  -e N_CONSUMERS=1 -e SINGLE_CONTAINER_MODE=true -e CREATE_PRODUCER=true -e INMEMORY_WORKERS=true -p 8080:80 --add-host=host.docker.internal:host-gateway --name test_backend openmined/grid-backend:local-dev'
```

Comment:
suggestion: port number can be derived from env

Reply:
Resolved in [c03f06f](https://github.com/OpenMined/PySyft/pull/8777/commits/c03f06fb6c336b57ffbada66345c63ce3644c808)